### PR TITLE
[Bug fix] Fix the case when GNN encoder is not used.

### PR DESCRIPTION
**Pull request by classicsong**
When GNN encoder is not used, node and edge classification/regression task will fail due to empty blocks returned by dataloader.

#37 

*Description of changes:*
Fix the bug

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: tests/unit-tests/data_utils.py [hunk 1]
```diff
@@ -156,6 +162,66 @@ def generate_dummy_dist_graph(dirname, size='tiny', graph_name='dummy'):
     return partion_and_load_distributed_graph(hetero_graph=hetero_graph, dirname=dirname,
                                               graph_name=graph_name)
 
+def create_lm_graph(tmpdirname):
```
Comment (zheng-da) [inline tests/unit-tests/data_utils.py:165]: can you add docstring to explain the difference between the two LM graphs?

File: tests/unit-tests/test_embed.py [hunk 2]
```diff
@@ -203,75 +204,15 @@ def test_compute_embed(dev):
...
-        input_ids1, attention_mask1, g
-
 def test_lm_infer():
     # initialize the torch distributed environment
     th.distributed.init_process_group(backend='gloo',
                                       init_method='tcp://127.0.0.1:23456',
                                       rank=0,
                                       world_size=1)
-    lm_config, feat_size, input_ids, attention_mask, g = create_lm_graph()
+    with tempfile.TemporaryDirectory() as tmpdirname:
```
Comment (zheng-da) [inline tests/unit-tests/test_embed.py:213]: is it right that the temporary file will no longer be needed after the graph is constructed?

File: tests/unit-tests/test_gnn.py [hunk 3]
```diff
@@ -93,6 +96,19 @@ def check_node_prediction(model, data):
     assert_almost_equal(pred1[0:len(pred1)].numpy(), pred2[0:len(pred2)].numpy(), decimal=5)
     assert_equal(labels1.numpy(), labels2.numpy())
 
+def check_mlp_node_prediction(model, data):
```
Comment (zheng-da) [inline tests/unit-tests/test_gnn.py:99]: maybe our test should explain why we want to have this test case.

File: tests/unit-tests/test_embed.py (hunk 2 above)
Comment (classicsong) [reply tests/unit-tests/test_embed.py:213]: Once we have g and lm_config, we do not need the graph files.

File: tests/unit-tests/data_utils.py (hunk 1 above)
Comment (classicsong) [reply tests/unit-tests/data_utils.py:165]: Updated.

File: tests/unit-tests/test_gnn.py (hunk 3 above)
Comment (classicsong) [reply tests/unit-tests/test_gnn.py:99]: Agree.
I put some docstring to tests in test_gnn.py

Comment (zheng-da) [reply tests/unit-tests/test_gnn.py:99]: i don't see docstring in this function.

**Review by zheng-da — APPROVED** [review]

**Pull request merged**